Source/OxyPlot.GtkSharp/OxyPlot.GtkSharp.csproj

```diff
@@ -12,7 +12,7 @@
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <ProductVersion>10.0.0</ProductVersion>
+    <ProductVersion>8.0.30703</ProductVersion>
```

Comment:
Yes.  Not sure the best way to resolve this sort of thing.   Unfortunately whatever visual studio or mono dev is used will override with its own meta data.

I did adjust the project references though.  They were pointing to an explicit path in c:\Program Files (x86) rather than self-referring to OxyPlot projects.